### PR TITLE
Fix: Require friendsofphp/php-cs-fixer

### PR DESCRIPTION
**Pull request by localheinz**
This PR
- [x] requires `friendsofphp/php-cs-fixer`
- [x] adds composer script to ease local development

💁 If we have a [configuration](https://github.com/thephpleague/iso3166/blob/master/.php_cs), then we could just as well use it, right?

### CODE REVIEW

**Comment by alcohol** [comment]
I'd rather not make this part of the build process. I prefer to run this manually :-)

**Comment by localheinz** [comment]
@alcohol 

Shall I remove https://github.com/thephpleague/iso3166/pull/16/commits/8e9b816a02ff0bb72379cc4c6d5ae58326420364 then?

**Comment by alcohol** [comment]
Yea, and if you could add `test` (or `phpunit`) as a command as well next to `cs` that'd be great :-)

File: composer.json [hunk 1]
```diff
@@ -32,5 +33,9 @@
         "branch-alias": {
             "dev-master": "1.x-dev"
         }
+    },
+    "scripts": {
+        "cs": "php-cs-fixer fix --verbose --diff",
+        "test": "phpunit"
```
Comment (localheinz) [inline composer.json:39]: @alcohol 

Worth adding a `.github/CONTRIBUTING.md` with notes about this, and referencing it from `README.md`?

Comment (alcohol) [inline composer.json:39]: Well there is also https://iso3166.thephpleague.com/contributing already

**Pull request merged**